### PR TITLE
feat: Error handling if post case fails

### PR DESCRIPTION
**Pull request by mikaelaalu**
Can't be merged until this is merged https://github.com/allmannyttan/slussen/pull/61

This PR:
- Show loading component on frontend while waiting for response from slussen

Left to do:
- Need to fix better error handling for frontend when receiving status code 400

### CODE REVIEW

File: packages/client/src/pages/Summary.tsx [hunk 1]
```diff
@@ -1,47 +1,50 @@
...
   const [completeErrorReport] = useAtom(reportAtom)
+  const [loading, setLoading] = React.useState(false)
 
   const submit = async () => {
     const formdata = new FormData()
     formdata.append('object', completeErrorReport.object)
     formdata.append('place', completeErrorReport.place)
     formdata.append('room', completeErrorReport.room)
     formdata.append('area', completeErrorReport.area)
+    formdata.append('rentalId', '12345')
```
Comment (mull) [inline packages/client/src/pages/Summary.tsx]: @izabellalarsson vet du vilket värde som ska användas här?

Comment (izabellalarsson) [reply packages/client/src/pages/Summary.tsx]: ja! det värdet finns i userData 
`const [userData] = useAtom(userAtom)`

**Review by izabellalarsson — APPROVED** [review]

File: docker-compose.yml [hunk 2]
```diff
@@ -45,24 +45,24 @@ services:
...
-      - "FAST_API__BASE_URL=${FASTAPIBASEURL:-http://www.fastapi.se/backendprop/v1/api/}"
-      - "FAST_API__ACCESS_TOKEN=${FASTAPIACCESSTOKEN:-12345}"
-      - "FAST_API__USER=${FASTAPIUSER:-demouser}"
-      - "FAST_API__PASSWORD=${FASTAPIPASSWORD:-demopassword}"
-      - "DATABASE_URL=${DATABASE_URL:-postgres://postgres:postgrespassword@slussen-postgres:5432/api-db}"
-      - "POSTGRES__HOST=${POSTGRESHOST:-slussen-postgres}"
-    restart: on-failure
-    depends_on:
-      - slussen-postgres
+  # slussen:
```
Comment (izabellalarsson) [inline docker-compose.yml]: oops

Comment (izabellalarsson) [reply docker-compose.yml]: bring back :)

File: packages/server/src/config.ts [hunk 3]
```diff
@@ -28,9 +28,9 @@ const config = configPackage({
       maxFailedLoginAttempts: 3,
     },
     api: {
-      baseUrl: 'http://slussen:4000',
-      username: 'test',
-      password: 'kalas',
+      baseUrl: 'http://host.docker.internal:4000',
+      username: 'mull',
+      password: 'shanas',
```
Comment (izabellalarsson) [inline packages/server/src/config.ts]: here as well

**Pull request merged**